src/libs/modulegroups.c

```diff
@@ -2153,6 +2153,7 @@ static void _manage_editor_save(dt_lib_module_t *self)
   // update the preset in the database
   dt_lib_presets_update(d->edit_preset, self->plugin_name, self->version(), newname, "", params, strlen(params));
   g_free(params);
+  d->edit_preset = g_strdup(newname);
```

Comment:
don't we need to free d->edit_preset first ?

Reply:
indeed we do